### ISSUE TITLE
Can't set environmentvariables in install script

### ISSUE THREAD

**Issue by maldun** [lutris/lutris#1715]
Lutris version is 0.4.23 on fedora 29

For the game [SK Burst Re:Newal](https://lutris.net/games/senran-kagura-burst-renewal/) I wanted to let the game install via steam on linux with steamplay enabled.
Since I need PROTON_NO_ESYNCH=1 set I tried to set it via ENV variables by adding 
```python
system:
    env: 
        PROTON_NO_ESYNCH: '1'
```
but if I execute in the shell, I don't find the environment variable when typing env.
It surely is strange ... is there also another way to provide shell variables during a game run?

**Comment by strycore** [comment]
No enough info, should work, closing.

**Comment by maldun** [comment]
> No enough info, should work, closing.

I have now retried it with the attached install script. In the console PROTON_NO_ESYNCH is still not set to 1
[senran-kagura-burst-renewal-steam-proton.zip](https://github.com/lutris/lutris/files/2998179/senran-kagura-burst-renewal-steam-proton.zip)

**Comment by strycore** [comment]
it's PROTON_NO_ESYNC, not PROTON_NO_ESYNCH

**Comment by maldun** [comment]
> it's PROTON_NO_ESYNC, not PROTON_NO_ESYNCH

Thanks for pointing that out, you're right, but I just hacked this .json together since I lost my original files some time ago, so I made this typo ...
Nevertheless I can't get the PROTON_NO_ESYNC into the environment (same held for the wrongly written PROTON_NO_ESYNCH). I attach the corrected file and the env of the bash which is started by Lutris.
[files.zip](https://github.com/lutris/lutris/files/2998853/files.zip)

Thanks for your patience!
